syntax = "proto3";

package NUgc;

message TReviewMeta {
    string YandexTLD = 1;
    string CountryId = 2;
    string UpdateId = 3;
};
